src/components/CatalogOverlay/CatalogPlotComponent/CatalogPlotComponent.tsx

```diff
@@ -604,8 +604,8 @@ export class CatalogPlotComponent extends React.Component<WidgetProps> {
         const difference = range[1] - range[0];
```

Comment:
should we do Number(range[1]) - Number(range[0]) as well?

Reply:
The difference between two "number" strings will be a number in JS. But we should convert the string to a number earlier.